scanpy/tools/_dendrogram.py

```diff
@@ -156,7 +156,10 @@ def dendrogram(
 
     if inplace:
         if key_added is None:
-            key_added = f'dendrogram_{groupby}'
+            if len(groupby) == 1:
+                key_added = f'dendrogram_{groupby[0]}'
+            else:
+                key_added = f'dendropgram_{"_".join(groupby)}'
```

Comment:
dendropgram had a typo.

Reply:
Is groupby with more than one element possible?

Reply:
Oops, fixed the typo. That's what I get for editing code in a browser. I'll try to add a test for this.

I think the `sc.tl.dendrogram` function allows you to pass a list of groupby variables. I haven't used it myself, but I imagine there's a use case for it?

Reply:
I've added a test for this now, and confirmed locally that the test passes.